### PR TITLE
Modify the code to support the 2201.10.x lang version

### PR DESCRIPTION
**Pull request by Nuvindu**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,17 +1,16 @@
 org.gradle.caching=true
 group=io.ballerina.lib
 version=0.1.2-SNAPSHOT
```
Comment (NipunaRanasinghe) [inline gradle.properties:3]: Don't we need to bump the module version if we are planning to do a new release on top of U10?

**Review by NipunaRanasinghe — DISMISSED** [review]
Approved with a minor suggestion

File: gradle.properties (hunk 1 above)
Comment (Nuvindu) [inline gradle.properties:3]:
```suggestion
version=0.2.2-SNAPSHOT
```

File: gradle.properties [hunk 2]
```diff
@@ -1,17 +1,16 @@
 org.gradle.caching=true
 group=io.ballerina.lib
-version=0.1.2-SNAPSHOT
-ballerinaLangVersion=2201.8.6
+version=0.2.2-SNAPSHOT
```
Comment (Nuvindu) [inline gradle.properties:3]:
```suggestion
version=0.2.0-SNAPSHOT
```

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**